src/web/handlers.rs

```diff
@@ -44,7 +44,12 @@ pub async fn get_tiles(
         let addr = match ip_addr_str.parse() {
             Ok(v) => v,
             Err(e) => {
-                return Err(HandlerErrorKind::General(format!("Invalid remote IP {:?}", e)).into());
+                // Temporary: log the IP addr for debugging mmdb issues
+                return Err(HandlerErrorKind::General(format!(
```

Comment:
Since IPs are relatively sensitive information, should this be a configurable option that defaults to off? We could then turn it back off without having to cut a new release? (Or is there some other plan that isn't linked to from here?)

Reply:
I can but I'm not intending for this to be used outside of the dev env.

I'm working on a change for #148 that will land shortly where I'll be intending to revert his